hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -68,12 +68,12 @@
     private final long timeToLiveMillis;
     private final boolean invalidateOnChange;
     private final EvictionPolicy evictionPolicy;
-    private final AtomicBoolean canCleanUp;
+    private final AtomicBoolean canExpire;
```

Comment:
`ClientHeapNearCache` and `NearCacheImpl` are candidates to be removed soon. I just made a little naming adaption to have a better picture of eviction vs. expiration. I won't put much effort in these classes anymore.